recipe/meta.yaml

```diff
@@ -47,13 +47,14 @@ test:
     - conda inspect objects freetds  # [osx]
 
 about:
-  home: http://www.freetds.org/
-  license: LGPL-2.0 AND GPL-2.0
-  license_file: 
+  home: https://www.freetds.org/
+  license: LGPL-2.0
+  license_file:
     - COPYING_LIB.txt
```

Comment:
why do we not need to include both COPYING.txt and COPYING_LIB.txt?

Reply:
`COPYING_LIB.txt` contains the text of the LGPLv2 license while `COPYING.txt` is GPLv2. The project page states that they use an LGPLv2 license.

* https://github.com/FreeTDS/freetds#readme-for-freetds-14dev-development
* https://www.freetds.org/faq.html#license